runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -180,6 +192,17 @@ func (p *Parser) parseMetricsView(ctx context.Context, node *Node) error {
 		return fmt.Errorf("must define at least one measure")
 	}
 
+	tmp.DefaultComparison.Mode = strings.ToLower(tmp.DefaultComparison.Mode)
+	if _, ok := comparisonModesMap[tmp.DefaultComparison.Mode]; !ok {
+		return fmt.Errorf("invalid mode: %q. allowed values: %s",
+			tmp.DefaultComparison.Mode, strings.Join(validComparisonModes, ","))
```

Comment:
nit: just make this one line (we've generally been cool with wide lines on the backend, we have wide displays and space is more precious vertically; when we do break lines for readability, do it at every breakpoint for consistency (e.g. at every `,` in the scope))